config/recipes/autopilot/elasticsearch.yaml

```diff
@@ -30,7 +30,7 @@ spec:
           securityContext:
             privileged: true
             runAsUser: 0
-          command: ['/usr/local/bin/bash', '-e', '-c', 'echo 262144 > /proc/sys/vm/max_map_count']
+          command: ['/usr/local/bin/bash', '-e', '-c', 'echo 1048576 > /proc/sys/vm/max_map_count']
```

Comment:
My apologies, but after testing this new setting. I think we have to stick with the old value for now. GKE Autopilot has a whitelist for this DaemonSet that is based on the exact hash/contents. Which means we have to stick with the old value until we talked to Google about whitelisting the version with the new value as well.